daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,53 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $id =  (Get-WmiObject -Class Win32_ComputerSystemProduct).UUID.ToLower()
+  $edition = Get-MetadataValue -key 'edition'
+  $family = Get-MetadataValue -key 'family'
+  $name = Get-MetadataValue -key 'name'
+  $release_date = Get-Date -Format "yyyyMMdd"
+  $release_time = (Get-Date -UFormat "%s").substring(0, 10)
+  $image_metadata = @{'id' = $id,
+                      'family' = $family;
+                      'version' = $edition;
+                      'name' = $name;
+                      'location' = "";
+                      'release_date' = $release_date;
+                      'release_time' = $release_time;
+                      'state' = "Active";
+                      'environment' = "Prod";
+                      'packages' = @()}
+
+  # Get Googet packages.
+  $out = & 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' 'installed'
+  $out = $out[1..$out.length]
+  [array]::sort($out)
+
+  foreach ($package_line in $out) {
+    $split = $package_line.Trim() -split '\s+'
+    $id =  (Get-WmiObject -Class Win32_ComputerSystemProduct).UUID.ToLower()
+    $name = $split[0]
+    $version = $split[1]
+    # TODO: Currently, Installed command only return name and version. We need to update command to get all info.
+    $commit_hash = ""
+    $release_date = ""
```

Comment:
What is this supposed to be? package build date? Package release date doesnt make much sense.

Reply:
And these two field should populate from "googet installed" command. The command currently only return name and version from spec.

Reply:
We already put the build date in the version, and the git commit it's built off, so build date here doesn't add anything we don't ready know.